packages/encryption/src/lib/schemas/index.spec.ts

```diff
@@ -0,0 +1,73 @@
+import * as fs from 'fs';
```

Comment:
see other comments about moving implementation for schema validation outside of this package.